schema.graphql

```diff
@@ -544,6 +544,7 @@ type Mutation {
   unlikeComment(id: ID!): Comment
   unlikePost(id: ID!): Post
   unregisterForEventByUser(id: ID!): Event!
+  updateAdvertisement(input: UpdateAdvertisementInput!): Advertisement
```

Comment:
Make a new `type UpdateAdvertisementPayload` for defining shape of the mutation result. Refer to this https://github.com/PalisadoesFoundation/talawa-api/pull/1585#discussion_r1441534417